### ISSUE TITLE
Code completion missing primitive methods

### ISSUE THREAD

**Issue by cgburgess** [intellij-rust/intellij-rust#3358]
## Environment

Intellij-Rust plugin version: v0.2.91.2115-183
Rust toolchain version: 1.32
IDE name and version: CLion 2018.3.4 and Intellij Ultimate 2018.3.3
Operating system: Windows 10 64

## Problem description
Missing syntax completion and declaration go to on some (newer?) methods defined on the primitive types.

Methods in this example defined here: [u32 rust doc](https://doc.rust-lang.org/std/primitive.u32.html#method.from_le_bytes)

## Steps to reproduce
```rust
    let u32_example = u32::from_be_bytes([0, 0, 0, 1]);
    let u16_example = u16::from_le_bytes([0, 1]);
    println!("{} and {}", &u32_example, &u16_example);
```
1 and 256

picture of no highlighting or code completion. 
![example](https://user-images.githubusercontent.com/9002722/52393744-7856b780-2a75-11e9-9b5f-e60d62acd3a4.PNG)

**Comment by Undin** [comment]
@Dakati it's because the corresponding methods are implemented via macros
See #2458 for more details

**Comment by Undin** [comment]
Work with new macro expansion engine.
Fixed by #5716
See #3628 for more details
